recipes/mosquitto/1.x/conanfile.py

```diff
@@ -0,0 +1,132 @@
+import os
+import glob
+from conans import CMake, ConanFile, tools
+
+required_conan_version = ">=1.29.1"
+
+class MosquittoConan(ConanFile):
+    name = "mosquitto"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/eclipse/mosquitto"
+    topics = ("mqtt", "broker", "libwebsockets", "mosquitto", "eclipse", "iot")
+    license = "EPL-1.0", "BSD-3-Clause" # https://lists.spdx.org/g/Spdx-legal/message/2701
+    description = "Eclipse Mosquitto - An open source MQTT broker"
```

Comment:
I think meta (description, license, topics, homepage) should be the same for both versions

Reply:
The license actually changed

Reply:
Than we should open a PR against 2.x to add the missing items 🤷 Does the homepage to github vs their website make a huge difference?

https://github.com/conan-io/conan-center-index/blob/3f1dfa2419b23b0c2c2639c5278ed50fb4b3c435/recipes/mosquitto/2.x/conanfile.py#L13-L15